src/test/it/essentials.yml

```diff
@@ -0,0 +1,37 @@
+---
+packaging:
+  bom: bom.yml
+  environment: aws
+  archiveArtifacts: true
+  cwpVersion: 0.1-alpha-6
+  config:
+    bundle:
+      vendor: "Jenkins project"
+      title: "Jenkins WAR - Artifact Manager S3 Integration Tests"
+      description: "Demo build, which includes Artifact Manager S3 and plugins using ArtifactManager API for testing"
+    groovyHooks:
+      - type: "init"
+        id: "initScripts"
+        source:
+          dir: initScripts
+ath:
+  # TODO: ATH test dependency issues
+  disabled: true
```

Comment:
Why does this attribute need to exist? Why not just comment out the whole `ath` section?

Reply:
Yes, you can do that if you want